app/models/like.rb

```diff
@@ -0,0 +1,8 @@
+class Like < ApplicationRecord
+    belongs_to :post, foreign_key: 'post_id'
+    belongs_to :author, class_name: 'User', foreign_key: 'author_id'
+
```

Comment:
- [OPTIONAL] There is no need to use the `foreign_key` attribute to define the relationship here. Rails use conventions to create models, so it knows how to link posts as long as they are properly defined in the _**model**_ files. You can adopt the following code:

```ruby
belongs_to :post
belongs_to :author, class_name: 'User'
```